recipe/meta.yaml

```diff
@@ -34,10 +40,11 @@ requirements:
     - setuptools_scm >=6.2
     - tomli
     - py-cpuinfo
+    - numpy
   run:
     - python
-    - numpy >=1.7
     - msgpack-python
+    - {{ pin_compatible('numpy') }}
```

Comment:
This can be dropped

```suggestion
```

xref: https://github.com/conda-forge/conda-forge.github.io/issues/2156